checkpoint_service.py

```diff
@@ -0,0 +1,71 @@
+import os
+import logging
+
+# List of all objectTypes that we export / import in WM
+USER_OBJECT = "users"
+GROUP_OBJECT = "groups"
+WORKSPACE_NOTEBOOK_OBJECT = "notebooks"
+WORKSPACE_DIRECTORY_OBJECT = "directories"
+WORKSPACE_NOTEBOOK_ACL_OBJECT = "acl_notebooks"
+WORKSPACE_DIRECTORY_ACL_OBJECT = "acl_directories"
+METASTORE_TABLES = "metastore"
+
+# Actions
+WM_EXPORT = "export"
+WM_IMPORT = "import"
+
+class CheckpointKeySet():
+    """Class that deals with checkpoint read and write of a particular object type"""
+
+    """
```

Comment:
nit: No need to have a separate doc string for params? (may be we can merge them into one)

Reply:
done